lighthouse-core/test/report/html/renderer/base64-test.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Base64 = require('../../../../report/html/renderer/base64.js');
+
+/* eslint-env jest */
+
+describe('base64', () => {
+  /** @type {string} */
+  async function test(str) {
+    for (const gzip of [false, true]) {
+      // Already tested in treemap-test-pptr.js
+      // TODO: can we test this in Node?
```

Comment:
It'd be nice to at least get the decode gzip exercised on these edge cases. is it a lot of work to have `encode` fallback to `pako` if compression stream is missing?

Reply:
How about we remove `pako` from base64 entirely, but totally hack & define CompressionString in the unit test to use pako?

Reply:
actually i have no idea how to fake a stream or readable so nevvermind lets add pako as a fallback